### ISSUE TITLE
[Bug] The background of BottomNavigation is sometimes displayed out of alignment.

### ISSUE THREAD

**Issue by Corvus400** [DroidKaigi/conference-app-2024#551]
- When I try to enter text on the ProfileCardScreen, the background is misaligned as shown in the attached video.

https://github.com/user-attachments/assets/426dc257-bc51-4e05-853f-57ad5b85112b

**Comment by Corvus400** [comment]
I would like to try this task as it seems to be fixable after investigation.

**Comment by kosenda** [comment]
@takahirom @Corvus400 
This issue seems to be occurring again in the main branch ...！

**Comment by takahirom** [comment]
Let's reopen

**Comment by Corvus400** [comment]
@takahirom @kosenda 
We are trying to address this issue, but the problem is not reproducible again. 🥹 
In its current state, when the keyboard is raised, BottomNavigation is not visible, so I cannot see if the problem is happening. 🙈

**Comment by kosenda** [comment]
Indeed, with the latest version, the issue no longer occurs on my device as it is hidden by the keyboard!

**Comment by takahirom** [comment]
Thank you for confirming. Let's close this issue, and if it happens again, we'll reopen it.

**Comment by Corvus400** [comment]
@takahirom @kosenda 
We found that this problem recurred when we switched to Composable Multiplatform on iOS. 👀 
![スクリーンショット 2024-08-23 18 39 53](https://github.com/user-attachments/assets/d7e6d8bc-b9dd-4fd1-ac0b-790bb832012b)
